js/test/unit/net/ip4.js

```diff
@@ -108,16 +131,42 @@ test('receive ip4 fragmented overlapped fragments', (t) => {
 });
 
 test('receive ip4 fragmented overlapped fragments ladder 1', (t) => {
-  const slices = [
-    { offset: 8, len: 64 },   // [ ========]
-    { offset: 16, len: 56 },  // [  =======]
-    { offset: 24, len: 48 },  // [   ======]
-    { offset: 32, len: 40 },  // [    =====]
-    { offset: 40, len: 32 },  // [     ====]
-    { offset: 48, len: 24 },  // [      ===]
-    { offset: 56, len: 16 },  // [       ==]
-    { offset: 64, len: 8 },   // [        =]
-    { offset: 0, len: 8 },    // [=        ] (last piece)
+  const slices = [{
+    offset: 8,
+    len: 64,
+  }, // [ ========]
```

Comment:
looks ok, comment no longer readable though. Move it outside before the const?
or keep it as is if lint allows?